notebooks/dataframes/struct_and_array_dtypes.ipynb

```diff
@@ -0,0 +1,658 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    adj_scores_arr = bbq.array_agg(adj_scores.groupby(level=0))
</pre><p> </p><p>Could we add a note about ordering? Is the original order preserved? If so, under what conditions?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/googleapis/python-bigquery-dataframes/pull/883/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>